From c7a6bf126c17e997428ee925aad62b2fd70dc316 Mon Sep 17 00:00:00 2001
From: Dave Taht <dave.taht@bufferbloat.net>
Date: Fri, 13 Apr 2012 23:14:06 -0700
Subject: [PATCH] _XOPEN_SOURCE breaks CLOCK_REALTIME in uclibc

Since you can't tell what library you are using...
until after you include features.h
which after you would set _XOPEN_SOURCE
and if you set _XOPEN_SOURCE you can't re-include features.h
we have a recursive dependency.

I have no idea how to handle this, so I'll just carry a patch
in openwrt.
---
 gpsutils.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/gpsutils.c b/gpsutils.c
index 60c4ae9..95ab456 100644
--- a/gpsutils.c
+++ b/gpsutils.c
@@ -4,10 +4,6 @@
  * BSD terms apply: see the file COPYING in the distribution root for details.
  */
 
-/* The strptime prototype is not provided unless explicitly requested.
- *  So add the define that POSIX says to to avoid: */
-#define _XOPEN_SOURCE
-
 #include <stdio.h>
 #include <time.h>
 #include <sys/time.h>
-- 
1.7.5.4

